src/azure-cli/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -40,6 +40,17 @@ def zip_contents_from_dir(dirPath, lang):
                     subdirs.remove('bin')
                 elif 'obj' in subdirs:
                     subdirs.remove('obj')
+            elif lang.lower() == PYTHON_RUNTIME_NAME:
+                _folder_to_search = os.path.join('lib', 'site-packages')
+                path_val = dirname.lower()
+                if _folder_to_search in path_val:
+                    _parent_path = path_val[:path_val.index(_folder_to_search)]
+                    import shutil
+                    shutil.rmtree(_parent_path, ignore_errors=True)  # ignore errors so that we don't fail the operation
+            if '.github' in subdirs:
```

Comment:
Are you sure to remove this as this will detach the source from git source control. To me it is a data loss. Can we copy to a temp folder, prune and deploy from there?

Reply:
you are talking just about the .github check right, I will just remove that for now.